lib/ansible/module_utils/ecs/api.py

```diff
@@ -0,0 +1,353 @@
+# This code is part of Ansible, but is an independent component.
+# This particular file snippet, and this file snippet only, is licensed under the
+# Modified BSD License. Modules you write using this snippet, which is embedded
+# dynamically by Ansible, still belong to the author of the module, and may assign
+# their own license to the complete work.
+#
+# "Modified BSD License" (BSD-3 Clause License)
```

Comment:
Ah, one thing: the 3-clause BSD license can't be used for module_utils. You need to use the 2-clause BSD license.

The best way is probably to use the short header:
```
# Copyright (c), Entrust Datacard Corporation, 2019 (or something like that :) )
# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
```
(Similar to what https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py does)